test/minitest_helper.rb

```diff
@@ -173,7 +191,8 @@ def has_edge?(edge, traces)
 # against.
 #
 def valid_edges?(traces)
-  traces.reverse.each do  |t|
+  return true unless traces.is_a?(Array) # why do we need this?
```

Comment:
I don't know why it's needed either :-)

Reply:
I started switching my tests to report to the collector. In that case I don't have traces to inspect, but I didn't wanted my test suite to report exception, just failed tests. 
I should add a comment.